adcm_aio_client/core/objects/cm.py

```diff
@@ -221,8 +221,14 @@ class ClustersNode(PaginatedAccessor[Cluster]):
     filtering = Filtering(FilterByName, FilterByBundle, FilterByStatus)
 
     async def create(self: Self, bundle: Bundle, name: str, description: str = "") -> Cluster:
+        bundle_data = await self._requester.get("bundles", bundle.id)
```

Comment:
to get data for prototype id. I can't access bundle's _main_prototype_id from outside but i need to pass prototype id in order to create object. So either a method returning it or general data from bundle page

Reply:
Your motivation is valid

We've already encountered some places in client where we have conflict between what we "need" to access and what we shouldn't display to user

@a-alferov I suggest allowing internal usage in "our" code, because making separate requests in redundant AND we shouldn't make too much of "semi-public" API + it's too much to separate interface and implementation for now

Reply:
returned access to the protected